cluster/manifests/z-karpenter/vpa.yaml

```diff
@@ -19,6 +19,13 @@ spec:
     containerPolicies:
       - containerName: controller
         minAllowed:
-          memory: {{.Cluster.ConfigItems.karpenter_controller_memory}}
-          cpu: {{.Cluster.ConfigItems.karpenter_controller_cpu}}
+          memory: {{ .Cluster.ConfigItems.karpenter_controller_memory }}
+          cpu: {{ .Cluster.ConfigItems.karpenter_controller_cpu }}
+        maxAllowed:
+        {{ range $NodePool := .Cluster.NodePools}}
+        {{ if eq $NodePool.Name "default-master" }}
+          memory: {{ scaleQuantity (instanceTypeMemory (index .InstanceTypes 0)) 0.016 }}
+          cpu: {{ scaleQuantity (instanceTypeCPU (index .InstanceTypes 0)) 0.027 }}
```

Comment:
Let's add a comment explaining that this means 50m on `r6g.large` :)

Reply:
Added in the next commit. Thanks!